Rakefile

```diff
@@ -1,6 +1,10 @@
 require "yast/rake"
 
 Yast::Tasks.configuration do |conf|
+  conf.obs_api = "https://api.suse.de/"
+  conf.obs_target = "SLE-12-SP1"
+  conf.obs_sr_project = "SUSE:SLE-12-SP1:GA"
+  conf.obs_project = "Devel:YaST:Head"
```

Comment:
well, but it is not correct to overuse Head, which is master and will be SP2. It will hit us in future. Also currently there is under review imo change to introduce shortcut with `submit_to` - https://github.com/yast/yast-rake/pull/21